tests/conftest.py

```diff
@@ -212,6 +219,49 @@ def s3_rig(request, monkeypatch, assets_dir):
         bucket.objects.filter(Prefix=test_dir).delete()
 
 
+@fixture()
+def custom_s3_rig(request, monkeypatch, assets_dir):
+    drive = os.getenv("CUSTOM_S3_BUCKET", "bucket")
+    custom_endpoint_url = os.getenv("CUSTOM_S3_ENDPOINT")
+    custom_key_id = os.getenv("CUSTOM_S3_KEY_ID")
+    custom_secret_key = os.getenv("CUSTOM_S3_SECRET_KEY")
+    test_dir = create_test_dir_name(request)
+
+    # Upload test assets
```

Comment:
Let's keep the same `if os.getenv("USE_LIVE_CLOUD") == "1":` logic as in the S3 one so that local tests are fully mocked (`endpoint_url` is basically ignored in the mock), but we are running the integration tests against live backends we do this copying.

Reply:
done here fc113dc